### PR TITLE
CMA prototyping progression

### PR DESCRIPTION
**Pull request by jdillon**
related:

https://github.com/sonatype/nexus-pro/pull/487

### CODE REVIEW

File: plugins/basic/nexus-repository-nuget/src/main/java/com/sonatype/nexus/repository/nuget/internal/NugetFeedHandler.java [hunk 1]
```diff
@@ -45,7 +45,7 @@ public Response handle(@Nonnull final Context context) throws Exception {
 
     final String base = getRepositoryUri(context);
 
-    final String feed = facet.feed(base, operation, context.getRequest().getParameters());
+    final String feed = null; //facet.feed(base, operation, context.getRequest().getParameters());
```
Comment (kellyrob99) [inline plugins/basic/nexus-repository-nuget/src/main/java/com/sonatype/nexus/repository/nuget/internal/NugetFeedHandler.java]: still TBD?

File: plugins/rapture/nexus-coreui-plugin/src/main/java/org/sonatype/nexus/coreui/SearchResultVersionXO.groovy [hunk 2]
```diff
@@ -44,7 +45,7 @@ implements Comparable<SearchResultVersionXO>
     if (result == 0) {
       result = versionScheme.parseVersion(version ?: "0").compareTo(versionScheme.parseVersion(o.version ?: "0"))
       if (result == 0) {
-        result = path.compareTo(o.path)
+        result = path ?: "".compareTo(o.path ?: "")
```
Comment (kellyrob99) [inline plugins/rapture/nexus-coreui-plugin/src/main/java/org/sonatype/nexus/coreui/SearchResultVersionXO.groovy]: not sure this is logically equivalent for the first usage, if path is non-null you're not calling compareTo at all and setting result(int) to a String. Second usage achieves intention of avoiding null comparison

File: plugins/basic/nexus-repository-nuget/src/main/java/com/sonatype/nexus/repository/nuget/internal/NugetFeedHandler.java (hunk 1 above)
Comment (mrprescott) [inline plugins/basic/nexus-repository-nuget/src/main/java/com/sonatype/nexus/repository/nuget/internal/NugetFeedHandler.java]: This PR includes all of the ongoing CMA work, so yeah, this is half-written NuGet code.

**Comment by adreghiciu** [comment]
manually merged

**Pull request closed without merging**